contracts/pcv/uniswap/UniswapPCVDeposit.sol

```diff
@@ -155,17 +155,26 @@ contract UniswapPCVDeposit is IUniswapPCVDeposit, PCVDeposit, UniRef {
         5. rETH = sqrt(k / P)
         
         and rFEI = k / rETH by 1.
+
+        Finally scale the resistant reserves by the ratio owned by the contract
      */
     function resistantBalanceAndFei() public view override returns(uint256, uint256) {
-        (uint256 feiInPool, uint256 ethInPool) = getReserves();
-        Decimal.D256 memory ethFei = readOracle();
+        (uint256 feiInPool, uint256 otherInPool) = getReserves();
+
+        Decimal.D256 memory price = readOracle();
+
+        uint256 k = feiInPool * otherInPool;
 
-        // resistant eth/fei in pool
-        Decimal.D256 memory k = Decimal.from(feiInPool).mul(ethInPool);
-        uint256 resistantEthInPool = k.div(ethFei).asUint256().sqrt();
-        uint256 resistantFeiInPool = k.div(resistantEthInPool).asUint256();
+        // resistant other/fei in pool
+        uint256 resistantOtherInPool = Decimal.one().div(price).mul(k).asUint256().sqrt();
 
-        return (resistantEthInPool, resistantFeiInPool);
+        uint256 resistantFeiInPool = Decimal.ratio(k, resistantOtherInPool).asUint256();
+
+        Decimal.D256 memory ratioOwned = _ratioOwned();
```

Comment:
I haven't used the decimal library enough to know this - is this actually a ratio or is this a percentage? Just want to make sure we aren't multiplying by >100% of the token values.

Reply:
Its a percentage. Ratio as in fraction